src/dds/rs-dds-device-proxy.cpp

```diff
@@ -312,10 +312,14 @@ dds_device_proxy::dds_device_proxy( std::shared_ptr< const device_info > const &
         // The get_stream_profiles() call will initialize the profiles (calling dds_sensor_proxy::init_stream_profiles())
         for( auto & profile : sensor_proxy->get_stream_profiles() )
         {
-            auto & source_profiles = sensor_proxy->_formats_converter.get_source_profiles_from_target( profile );
-            if( source_profiles.size() != 1 )
-                LOG_ERROR( "More than one source profile available for [" << profile << "]: " << source_profiles );
-            auto source_profile = source_profiles[0];
+            auto source_profile = profile;
```

Comment:
Can you explain this change?

Reply:
We had a problem when requesting "raw" profiles, i.e. no format conversions.
Calling `_formats_converter.get_source_profiles_from_target` for raw profiles fails. No need to map back sensor profiles to their original source profiles.